bin/dry-run.bash

```diff
@@ -13,7 +13,7 @@ here="$(dirname "$(readlink -f "$0")")"
 # shellcheck source=bin/common.bash
 source "${here}/common.bash"
 
-config_load "$1"
+# config_load "$1"
```

Comment:
I skipped it here since `dry-run` will call into `ops helmfile` which will load the config. Although, I can't see either of them actually using the config so it has mostly been used as a validation step, and two checks felt unnecessary on a single `dry-run`.